src/index/ivf/ivf.cc

```diff
@@ -846,6 +847,7 @@ IvfIndexNode<faiss::IndexIVFFlat>::Serialize(BinarySet& binset) const {
             faiss::write_index(index_.get(), &writer);
             LOG_KNOWHERE_INFO_ << "write IVF_FLAT, file size " << writer.tellg();
         }
+        writer.addtailer(Type(), this->version_);
```

Comment:
Can move the `addtailer` to index.cc?  Just a suggestion, maybe different to index ser/des with file.